charts/consul/templates/crd-peeringacceptors.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.connectInject.enabled .Values.global.peering.enabled }}
+{{- if .Values.controller.enabled }}
```

Comment:
We should revert this change -- unfortunately the make ctrl-manifests script does this but for now we just have to reverse it.

Reply:
Thanks!

I will get back to you on those questions

Reply:
I have revert the change